Controller/Api/ContentController.php

```diff
@@ -9,8 +9,8 @@
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 
 use Opifer\ContentBundle\Model\ContentInterface;
-use Opifer\ContentBundle\Event\ContentResponseEvent;
-use Opifer\ContentBundle\Event\ResponseEvent;
+use Opifer\ContentBundle\EventListener\ContentResponseEvent;
+use Opifer\ContentBundle\EventListener\ResponseEvent;
```

Comment:
These are not `EventListeners`. These are the actual Events, so they should be in the `Event` namespace IMO...